### PR TITLE
Design1

### PR DESCRIPTION
**Pull request by catalinac3**
- created a day-bar.css file
- added respective divs to html in the footer of the card
- change the placeholder message in the input, to account for cities with same names in different countries (try rome, and rome,it)
- coded in js the function dayProgressBar  to control dimensions and fill of the progress bar created in the footer of the card

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -53,7 +55,7 @@ <h1 class="card-title">
                   id="city-input"
                   type="text"
                   class="form-control"
-                  placeholder="city"
+                  placeholder="city or city,country-code"
```
Comment (MrTim) [inline index.html]: I think just "city or country code".

File: jsFiles/design.js [hunk 2]
```diff
@@ -75,3 +75,79 @@ function skyColor(sunrise, sunset, offsetTime) {
...
+    currentDateUserLocation,
+    offsetTime,
+    true
+  );
+
+  const sunriseInMinute = timeConversion(sunrise, offsetTime, true);
+  const sunsetInMinutes = timeConversion(sunset, offsetTime, true);
+  const timeInMinDay = sunsetInMinutes - sunriseInMinute;
+
+  const timeInMinAfterSunset = 1440 - sunsetInMinutes;
```
Comment (MrTim) [inline jsFiles/design.js]: I think you can put it in a constant: `const minsInDay = 60 * 24;` then it's a bit easier to read the code.

**Review by MrTim — APPROVED** [review]
Fine for me! Just couple minor suggestions in the code.

File: index.html (hunk 1 above)
Comment (catalinac3) [reply index.html]: It is rome (city in the united states) or rome,it (city in Italy) , country code alone wouldn't be found.

Comment (catalinac3) [reply index.html]: maybe I could write an example : city (ie. rome or rome,it)

**Pull request merged**